### ISSUE TITLE
vtk_output with paraview

### ISSUE THREAD

**Issue by demetrafabris** [KratosMultiphysics/Kratos#8706]
**Description**
When I open my vtk_output files with paraview I can not see the structure which I'm studying (bridge deck)

**Scope**
See output results

**Expected behavior**
I should see pressure and velocity trend.

**Environment**
- OS: Ubuntu 20.10
- GiD14
- Python 3.8.6
- ParaView 5.7.0

**Comment by philbucher** [comment]
Hi @demetrafabris 

Can you explain the issue a bit more?
Also can you post your ProjectParameters?

**Comment by demetrafabris** [comment]
I can not load the folder (or the compress one .zip file) because it's too heavy.

[ProjectParametersCustom.txt](https://github.com/KratosMultiphysics/Kratos/files/6420022/ProjectParametersCustom.txt)

If I open the vtk files with ParaView on Windows i can see the model with pressure and velocity trend, but I obtained dome warnings (first picture). Thus, I was trying to open the same file on Ubuntu, but i can not see anything as you can see in the second picture loaded.

![ParaView_Windows](https://user-images.githubusercontent.com/81411661/116982527-3014b700-acc9-11eb-924b-9d3113ae8e6d.png)
![ParaView_Ubuntu](https://user-images.githubusercontent.com/81411661/116982543-34d96b00-acc9-11eb-9c99-ca3e7e4f8382.png)

**Comment by philbucher** [comment]
hm seems to be the same problem as #8640
can you try with `binary` as `file_format` in Win?
regarding linux, it seems like you didn't click the "Apply" button?

FYI @adityaghantasala

**Comment by adityaghantasala** [comment]
Hi It would be really helpful if we can have the .vtk file. Just one file is enough. (in ascii format)
so we can tryout even here on our machines. may be you can upload this file on gigamov  and send us the link ?

**Comment by demetrafabris** [comment]
I'm not allowed to load file in ascii format so I loaded it in txt format. Let me know if it can be useful, please.

Actually I don't know what gigamov is

[fluid_computational_model_part_0_1.txt](https://github.com/KratosMultiphysics/Kratos/files/6420191/fluid_computational_model_part_0_1.txt)

**Comment by demetrafabris** [comment]
> hm seems to be the same problem as #8640
> can you try with `binary` as `file_format` in Win?
> regarding linux, it seems like you didn't click the "Apply" button?
> 
> FYI @adityaghantasala

Even if I click the Apply button nothing changes

**Comment by adityaghantasala** [comment]
@demetrafabris  thank you for the file. 
I opened it on my machine. No warnings ! 
I am using Ubuntu, paraview 5.8.1

**Comment by adityaghantasala** [comment]
you uploaded the step number 1. 
Can you may be upload the file from the latest step. may be that has problems.

**Comment by demetrafabris** [comment]
[fluid_computational_model_part_0_2001.txt](https://github.com/KratosMultiphysics/Kratos/files/6421276/fluid_computational_model_part_0_2001.txt)

**Comment by adityaghantasala** [comment]
Hi ... I tried this also. 
Again .. no problems. No warnings even !

**Comment by demetrafabris** [comment]
Can it be a problem of my ParaView version? I've got ParaView 5.7.0

**Comment by philbucher** [comment]
can you try with a newer version?
I don't see any reason it shouldn't work with 5.7 but just to double check

**Comment by adityaghantasala** [comment]
ya I think so too. I used 5.6 and 5.5 versions also  ! (previously)

**Comment by demetrafabris** [comment]
Yes, I can. I'm downloading a newer version because with 5.7 version I still don't see nothing, as you can see
![Screenshot from 2021-05-04 15-52-53](https://user-images.githubusercontent.com/81411661/117014197-dc1dc880-acf0-11eb-8f71-c3f0163bfeee.png)

**Comment by philbucher** [comment]
![image](https://user-images.githubusercontent.com/25484702/117015670-3c613a00-acf2-11eb-9288-be297827f08d.png)

no clue what is happening

I downloaded 5.7 and it works flawless (Ubuntu 20.04)

I am quite confident this is not an issue with the vtk file, I checked it a bit.

Unfortunately I also don't have other ideas how to proceed
Any other idea @adityaghantasala ?

**Comment by demetrafabris** [comment]
I don't know if it may useful, but here there is the link to download the full vtk_output folder

https://we.tl/t-op2k62ida6

**Comment by demetrafabris** [comment]
I solved using ParaView 5.8.1
Now I can see my model and pressure and velocity trend
The only difference is that on Windows I read 57 m/s as maximum velocity (as can be seen in your image above), but now I red 32 m/s as maximum velocity.
It's quite strange, isn't it?

![Screenshot from 2021-05-05 22-52-00](https://user-images.githubusercontent.com/81411661/117207730-90ecde00-adf4-11eb-9def-d6a761b51317.png)

**Comment by philbucher** [comment]
> I solved using ParaView 5.8.1
Now I can see my model and pressure and velocity trend

cool! still strange that it wasn't working with the other one

did you try the binary format? I would anyway recommend using it as the files will be smaller

> The only difference is that on Windows I read 57 m/s as maximum velocity (as can be seen in your image above), but now I red 32 m/s as maximum velocity.
It's quite strange, isn't it?

I think you didn't rescale the values in the second picture, can you try?

**Comment by demetrafabris** [comment]
Ok, you are right. I didn't know it was possible to do it.
Excuse the perhaps ignorant question: so I should scale the wind speed at each time step? What I read in the legend is not the maximum speed in the total time analyzed?

**Comment by philbucher** [comment]
it depends on what you want
you can use "Rescale to data range over all timesteps" if you want the max values and a constant legend
